Sprint-2/errors/1.js

```diff
@@ -3,11 +3,14 @@
 // Why will an error occur when this program runs?
 // Try playing computer with the example to work out what is going on
 
+// Firstly, the decimalNumber variable has already been declared. Secondly, the function convertToPercentage has not been called correctly, instead console.log was used to log decimalNumber. By removing the const keyword and calling the function correctly allows the code to convert the given decimalNumber to a percentage.
+
 function convertToPercentage(decimalNumber) {
-  const decimalNumber = 0.5;
+  decimalNumber = 0.5;
```

Comment:
If you set the value of `decimalNumber` here, then the function will always return "50%", regardless of the value of the parameter.

Reply:
Rectified the code by not setting a value for decimalNumber so that the function can return different values as passed into the parameter